src/journal/cat.c

```diff
@@ -91,6 +97,15 @@ static int parse_argv(int argc, char *argv[]) {
                                                        "Failed to parse priority value.");
                         break;
 
+                case ARG_STDERR_PRIORITY:
+                        arg_stderr_priority = log_level_from_string(optarg);
+                        if (arg_stderr_priority < 0) {
+                                log_error("Failed to parse stderr priority value.");
+                                return -EINVAL;
+                        }
```

Comment:
please write this as:

```c
if (arg_stderr_priority < 0) 
        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Failed to parse stderr priority value: %s", optarg);
```

Reply:
Ah, good catch.  Fixed in the updated version of this PR.